testutil/testclient/testqueryclients/appquerier.go

```diff
@@ -0,0 +1,86 @@
+package testqueryclients
+
+import (
+	"context"
+	"testing"
+
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/golang/mock/gomock"
+
+	"github.com/pokt-network/poktroll/testutil/mockclient"
+	"github.com/pokt-network/poktroll/x/application/types"
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+// addressApplicationMap is a map of application addresses that are deemed to
+// exist on chain, if an address is not in this map an error will be returned
+// from the mock ApplicationQueryClient's GetApplication method.
+// The array of strings is the addresses of the delegated gateways that the
+// application is supposed to have.
+var addressApplicationMap map[string][]string
+
+func init() {
+	addressApplicationMap = make(map[string][]string)
+}
+
+// NewTestApplicationQueryClient creates a mock of the ApplicationQueryClient
+// which allows the caller to call GetApplication any times and will return
+// an application with the given address.
+// The delegateeNumber parameter is used to determine how many delegated
```

Comment:
There is no `delegateeNumber` parameter

Reply:
This was previously there I refactored and forgot the comments